src/contract/event.rs

```diff
@@ -446,7 +444,7 @@ mod tests {
             .stream()
             .boxed()
             .next()
-            .immediate()
+            .wait()
```

Comment:
`immediate` no longer works probably because of internals in web3 changed.

Reply:
Hmm, is it just for this method? It would be nice to know what `web3` method exactly is always yielding.